android/src/main/java/com/pylon/RNSpokestack/RNSpokestackModule.java

```diff
@@ -1,13 +1,18 @@
 
 package com.pylon.RNSpokestack;
 
+import android.util.Log;
+import android.os.*;
```

Comment:
That's quite a few classes; need all of them?

Reply:
is there a way to figure out which class a class hidden by `@hide` is using to resolve itself?

Reply:
`@hide` is a javadoc thing, so i'd need a bit more info...if you're using Android Studio, though, you should be able to just take out this import and see which lines get lit up red in the rest of the file; those are the classes you need to import.

Reply:
weird, was getting an import error for `AsyncResult` which is not directly importable, but that appears to have vanished now.

Reply:
`AsyncResult` is a public class, so you should be fine there. That's a Spokestack thing, not Android.

Reply:
...d'oh. Looks like there's one in Android too that I wasn't aware of. That'd be your problem—those two were conflicting with the import-all syntax, but getting rid of the unnecessary Android one made your IDE happy.